bin/testcase.py

```diff
@@ -266,14 +272,21 @@ def validate_format(
             else:
                 data = ret.err
 
-            bar.part_done(
-                ret.status or expect_rejection,
-                message,
-                data=data,
-                warn_instead_of_error=warn_instead_of_error,
-            )
+            if expect_rejection:
+                bar.debug(
+                    message,
```

Comment:
Hmm, so this causes fewer lines to be printed for invalid cases right? Fine with me i suppose

Reply:
yes